wasm-engines/Dockerfile

```diff
@@ -53,11 +53,18 @@ ENV PATH=/root/.cargo/bin:$PATH
 
 RUN rustup default 1.31.0
 
+RUN apt install -y gcc-8
+RUN apt install -y g++-8
+RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 10
+RUN update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-8 10
+
```

Comment:
not really a complaint about this PR in particular:

It's really wonky to be installing gcc-8, uninstalling and then installing gcc-7, in the same container build (but necessary).

I've been thinking that this monolithic Dockerfile should be refactored into a ["benchmark environment"](https://github.com/jwasinger/benchmarking/blob/update-charts-updated/wasm-engines/Dockerfile.bench_env) docker image which contains common deps used by different engines and benchmarking scripts.  Then each engine should have its own docker image.

Not only would it make it easier to add new engines, but it would reduce time that it takes to review changes to Dockerfiles.  Because right now, I have to build every single wasm engine that we do benchmarks against in order to test this PR.